ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -214,7 +218,13 @@ const RunComponent = () => {
               <TableCell>Logs</TableCell>
               <TableCell>
                 <Button
-                  onClick={() => downloadLogs(ortRun.id)}
+                  onClick={() =>
+                    downloadLogs(
+                      ortRun.id,
+                      '',
+                      Object.keys(ortRun.jobs).concat('config').join(',')
```

Comment:
I would either add a code comment here explaining why this is done or extract it into a descriptive variable.